engine/include/cubos/engine/input/action.hpp

```diff
@@ -86,14 +63,12 @@ namespace cubos::engine
         /// @param justPressed Whether this action was just pressed.
```

Comment:
Not part of this PR, but pls fix the docs to `Sets whether this action was just pressed.`